src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,9 +1,18 @@
 package mate.academy
 
-fun greetUsers() {
-    // implement function
-}
-
 fun main() {
     greetUsers()
 }
+
+fun greetUsers() {
+    var flag = true
+    while (flag) {
+        println("Please enter a name (or press Enter to quit): ")
+        val name : String? = readLine()
+        if (name == "") {
```

Comment:
Please check this documentation
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/is-null-or-empty.html